packages/core/test/integration/replication.test.ts

```diff
@@ -81,13 +82,15 @@ describe(getTestDialectTeaser('Replication'), () => {
       expectReadCalls();
     });
 
-    it('should run read-only transactions on the replica', async () => {
-      await deps.sequelize.transaction({ readOnly: true }, async transaction => {
-        return deps.User.findAll({ transaction });
-      });
+    if (instance.dialect.supports.startTransaction.readOnly) {
```

Comment:
this test must run on all dialects, as the `readOnly` option is supported by all dialects

Reply:
Update now.